CMakeLists.txt

```diff
@@ -72,14 +72,13 @@ else (DOXYGEN_FOUND)
   message("Doxygen needs to be installed to generate the doxygen documentation")
 endif (DOXYGEN_FOUND)
 
-find_package(CUDA REQUIRED)
 enable_language(CUDA)
-
 find_package(GSL)
 find_package(LAPACK)
 find_package(Casacore REQUIRED)
 find_package(Boost REQUIRED)
 find_package(Eigen3 REQUIRED)
+find_package(RapidJSON REQUIRED)
```

Comment:
You are finding the package but the `RapidJSON::RapidJSON` target is not consequently used (it should be linked by the `LeapAccelerateCLI` target I believe). It probably works in the Travis builds because it all gets installed under `/usr`, but when building against a non-standard installation compilation would fail.

Reply:
This compiled due to rapidjson being a header only library in a common location. I'll update as it looks like this will fail if rapidjson isn't installed to a common location